<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stocks Investments</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>231</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Portfolio Input</string>
    </property>
    <widget class="QTableWidget" name="tbl_portfolio">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>261</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <column>
      <property name="text">
       <string>Stocks</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quant</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="txt_ticker">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>&lt;Ticker&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_quant">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>&lt;Quant&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_removeAll">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>330</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove All</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_generate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Efficient Frontier Chart</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dt_init">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDate|Qt::ImhPreferNumbers</set>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2020</year>
      <month>1</month>
      <day>3</day>
     </datetime>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::YearSection</enum>
    </property>
    <property name="displayFormat">
     <string>yyyy-M-d</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="date">
     <date>
      <year>2020</year>
      <month>1</month>
      <day>3</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_initDate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Init Date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dt_end">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2020</year>
      <month>12</month>
      <day>28</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy-M-d</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="date">
     <date>
      <year>2020</year>
      <month>12</month>
      <day>28</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_endDate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>End Date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_iter">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>&lt;500&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Iterations">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Iterations</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stockReturns">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stock Returns Chart</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionEfficient_Frontier"/>
    <addaction name="actionStocks_Comparizon"/>
    <addaction name="actionChart_Analysis"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionEfficient_Frontier">
   <property name="text">
    <string>Efficient Frontier</string>
   </property>
  </action>
  <action name="actionStocks_Comparizon">
   <property name="text">
    <string>Stocks Comparizon</string>
   </property>
  </action>
  <action name="actionChart_Analysis">
   <property name="text">
    <string>Chart Analysis</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
